### PR TITLE
Example 0.3

### PR DESCRIPTION
**Pull request by ab93**
* fix: move torch dependency away from main
* use pip as a dependency manager instead of poetry
* add threshold vtx

### CODE REVIEW

**Review by vigith — DISMISSED** [review]
should we update the document so users will know that they have to include `pytorch-lightining`?

File: examples/numalogic-simple-pipeline/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
-FROM python:3.10-slim-bullseye AS builder
```
Comment (vigith) [inline examples/numalogic-simple-pipeline/Dockerfile:1]: is this right.. this will work only for M1 mac, not for everyone.. right?

Comment (ab93) [reply examples/numalogic-simple-pipeline/Dockerfile:1]: It works the other way round though. Let me try in an intel mac

**Comment by ab93** [comment]
> should we update the document so users will know that they have to include `pytorch-lightining`?

Yes, let me update the readme.

**Review by vigith — APPROVED** [review]

**Pull request merged**